includes/Checker/Checks/Abstract_PHP_CodeSniffer_Check.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Checks\PHP_CodeSniffer_Check
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Checks;
+
+use WordPress\Plugin_Check\Checker\Check;
+use WordPress\Plugin_Check\Checker\Check_Result;
+use Exception;
+
+/**
+ * Check for running one or more PHP CodeSniffer sniffs.
+ *
+ * @since n.e.x.t
+ */
+abstract class Abstract_PHP_CodeSniffer_Check implements Check {
+
+	/**
+	 * List of allowed PHPCS arguments.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @var array
```

Comment:
```suggestion
	 * @since n.e.x.t
	 * @var array
```

There shouldn't be blank lines between `@since` and `@var` docs per WP core docs standards.